### ISSUE TITLE
[CLIFF] About the SMPL camera parameters output

### ISSUE THREAD

**Issue by mimiliaogo** [huawei-noah/noah-research#118]
Hi, what is the exact meaning of the SMPL camera parameters output? 
As your paper says, they are the root translations relative to the camera coordinate, and they can be transformed to perspective projection parameters. 
1. So does this mean the output of the SMPL vertices is in 'World Coordinate' ? and then we need to add a translation as 'extrinsic matrix' to convert them into 'Camera Coordinate'?
2. How is this formula formulated? Why can the z translation be calculated like that?
![image](https://user-images.githubusercontent.com/44743062/192214581-cde696c8-35b9-49f1-9ddf-8ba316dafea4.png)

Thank you so much!

**Comment by mimiliaogo** [comment]
Also, Does the global rotation mentioned in your paper refer to the root rotation with respect to the world coordinate or to the camera coordinate?

**Comment by mimiliaogo** [comment]
I see your supp, which is very clear. THank you!

**Comment by mimiliaogo** [comment]
I see your supp, and understand the above questions. 
However, in the below picture, if now we used the cropped image as camera coordinate(the original version), should the tx and ty all be zero? (because the predicted human mesh will be at the center of the bounding box)
![image](https://user-images.githubusercontent.com/44743062/192244503-a31fc03c-c74a-4b56-bfd9-f0d2bf2c94c1.png)

**Comment by haofanwang** [comment]
Is there any way to project SMPL joints into image plane as 2D keypoints in the demo?

**Comment by zhihaolee** [comment]
@mimiliaogo 
No, tx and ty are not always zeros. The center of the bbox is C, corresponding to E in the virtual plane, but the root of the human mesh is R, which causes the root translation tx=|RE|. 
In another word, the human root (the Pelvis joint) is not always at the center of the bbox.

**Comment by zhihaolee** [comment]
@haofanwang Refer to #119.
